qa/paqa_latency.c

```diff
@@ -289,20 +287,14 @@ static int paqaCheckMultipleSuggested( PaDeviceIndex deviceIndex, int isInput )
             finalLatency = streamInfo->outputLatency;
         }
         printf("          finalLatency = %6.4f\n", finalLatency );
-        /* For the default low & high latency values, expect quite close; for other requested
-         * values, at worst the next power-of-2 may result (eg 513 -> 1024) */
-        toleranceRatio = ( (i == 0) || (i == ( numLoops - 1 )) ) ? 0.1 : 1.0;
-        QA_ASSERT_CLOSE( "final latency should be close to suggested latency",
-                        streamParameters.suggestedLatency, finalLatency, (streamParameters.suggestedLatency * toleranceRatio) );
-        if( i == 0 )
-        {
-            previousLatency = finalLatency;
-        }
-    }
+        err = Pa_CloseStream( stream );
 
-    if( numLoops > 1 )
-    {
-        QA_ASSERT_TRUE( " final latency should increase with suggested latency", (finalLatency > previousLatency) );
+        QA_ASSERT_TRUE( "Latency should increase monotonically with the suggested latency.",
+                        finalLatency > (previousLatency  - 0.001));
```

Comment:
I regard any decreasing behavior, even due to rounding error, as a bug, therefore I suggest:
```suggestion
                        finalLatency >= previousLatency);
```